### PR TITLE
[5. PATCH api/review/review_id] allows you to increment the votes sco…

### PR DESCRIPTION
**Pull request by Proto-Will**
…re by whatever number is input

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -62,5 +62,57 @@ describe('app', () => {
...
+          return request(app)
+            .patch('/api/reviews/2')
+            .send(newReviewInfo)
+            .expect(200)
+            .then(({ body }) => {
+              const { review } = body;
+                  expect(review).toEqual(finalResult)
+            })
+        })
+    })
```
Comment (emilyb93) [inline __tests__/api.test.js]: Id like to see some error handling here!

What happens if we try to vote for non existent articles/ wrong data type for the id?
What about if the patch request body we send is malformed/missing keys/wrong data etc?

File: __tests__/api.test.js [hunk 2]
```diff
@@ -62,5 +62,57 @@ describe('app', () => {
             })
```
Comment (emilyb93) [inline __tests__/api.test.js]: Id like to see some error handling for this endpoint. What if we request an out of range review? What if we request a review with an id that isnt a number?

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Would like to see more consideration for error handling please

**Review by CatHoang — COMMENTED** [review]
Hi Will, good extra error handling tests, however I would also suggest looking at testing for wrong data type for the value if it is not a number?

**Review by CatHoang — APPROVED** [review]
Looks great Will, feel free to merge!

**Pull request merged**